moduleframework/mtf_scheduler.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# TODO: why?
+# Author Petr Sklenar psklenar@gmail.com
+#
+# Meta test family (MTF) is a tool to test components of a modular Fedora:
+# https://docs.pagure.org/modularity/
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import argparse
+
+import os
+import logging
+import sys
+import yaml
+import moduleframework
+import subprocess
+import tempfile
+import shutil
+
+from sys import argv
+
+from avocado.utils import process
+from avocado.core.app import AvocadoApp
+from avocado.core.parser import Parser
+
+logger = logging.getLogger("mtf")
```

Comment:
I would prefer to have it in common.py so we can use it in whole MTF https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/common.py

Reply:
I use common.print_debug + common.print_info,
(probably) I will create new issue request to modify common.py with better logging ability for these methods